src/main/liberty/config/server.xml

```diff
@@ -2,21 +2,29 @@
 
   <featureManager>
       <feature>microprofile-1.3</feature>
+      <feature>usr:opentracingZipkin-0.30</feature>
   </featureManager>
 
   <httpEndpoint httpPort="${default.http.port}" httpsPort="${default.https.port}"
       id="defaultHttpEndpoint" host="*" />
 
   <!-- Automatically expand WAR files and EAR files -->
   <applicationManager autoExpand="true"/>
-
+    
+  <opentracingZipkin host="${env.zipkinHost}" port="${env.zipkinPort}"/>
+  
   <webApplication name="customer" location="${app.location}"/>
-
-  <mpJwt
-     id="${env.jwtid}"
-     jwksUri="${env.jwksUri}"
-     issuer="${env.jwksIssuer}">
-  </mpJwt>
+           
+  <!--These values are defined in pom.xml-->
+    <mpJwt
+         id="${env.jwtid}"
+         jwksUri="${env.jwksUri}"
+         issuer="${env.jwksIssuer}">
+    </mpJwt>
+    
+    <administrator-role>
+      <user-access-id>user:${env.userRealm}/user</user-access-id>
```

Comment:
Is this variable `userRealm` defined ?

Reply:
Not yet, since this is only working properly locally, but I added this as a placeholder until we figure out how to get this property both locally and on Kube